ingestion/src/metadata/orm_profiler/orm/registry.py

```diff
@@ -27,13 +27,39 @@ class CustomTypes(TypeRegistry):
     UUID = UUIDString
 
 
+class Dialects(Enum):
+    Hive = b"hive"
+    Postgres = "postgresql"
```

Comment:
Created a new Enum where we can easily list down the schemes for each source. We base each entry in the Data Types for the services and the value we can tune it depending on the scheme. This simplifies the usage and allows for non-direct handling of the dialect names as with Postgres